src/core/abc/session.abc.ts

```diff
@@ -240,6 +240,10 @@ export abstract class WhatsappSession {
     throw new NotImplementedByEngineError();
   }
 
+  public setInfoAdminsOnly(id) {
```

Comment:
It should accept `value` parameter so we can extend it later to allow disable "info admin only" .

No needs to add it to query, it'll be default behavior

```suggestion
  public setInfoAdminsOnly(id, value) {
```